RELEASES.md

```diff
@@ -6,48 +6,68 @@ Below are the versioning schemes used for publishing debs:
+| Ubuntu PRO to latest <series>-updates | `20.3+202004011202~ubuntu1~14.04.1` -> `20.4~ubuntu1~14.04.1` |
+| Ubuntu PRO to Daily PPA | `20.3+202004011202~ubuntu1~14.04.1` -> `20.4+202004021202~ubuntu1~14.04.1` |
 
 
 ## Devel Release Process
 
-Below is the procedure used to release ubuntu-advantage-client to an Ubuntu series:
+Below is the procedure used to release ubuntu-advantage-client to an Ubuntu series, it assumes we already have an annotated tag via `git tag -a 27.1` on the `release-27` branch which has not yet been uploaded to `pkg/ubuntu/devel`:
 
  1. git-ubuntu clone ubuntu-advantage-tools; cd ubuntu-advantage-tools
  2. git remote add upstream git@github.com:canonical/ubuntu-advantage-client.git
  3. git fetch upstream
- 4. git checkout upstream/release-24
- 5. git tag -a 24.3
- 6. git rebase --onto pkg/ubuntu/devel 24.2 24.3
- 7. git checkout -b pkg-upload-24.3
- 8. debuild -S
- 9. dput  ppa:chad.smith/ua-client-uploads ./ubuntu-advantage-tools_24.3_source.changes
- 10. create a release tarfile/changes file using uss-tableflip's build-package script
+ \# Assume last previous released tag on `release-27` branch was `27.0.2`
+ 4. git rebase --onto pkg/ubuntu/devel 27.0.2 27.1
+ 5. git checkout -B upload-27.1-impish # to create a new local branch name based on your detached branch in step 4
+ 6. git push `<your_launchpad_user>` upload-27.1-impish
+ 7. Create a PPA for upload reviewers at `https://launchpad.net/~<YOUR_LP_USER>/+activate-ppa`
+ 8. Push to your test PPA so upload reviewers can install easily test packages easily during review
+ ```bash
+    debuild -S
+    dput  ppa:<YOUR_LAUNCHPAD_USER>/ua-client-uploads ./ubuntu-advantage-tools_*_source.changes
 ```
- ./tools/make-release --series <SERIES> --ppa <PPA_URL_FOR_TEST_UPLOADS>
- # follow printed procedure for dput to test PPA, and tag the released commitish
-```
- 11. Create a merge proposal such as [24.2 PR](https://code.launchpad.net/~chad.smith/ubuntu/+source/ubuntu-advantage-tools/+git/ubuntu-advantage-tools/+merge/385073), [24.3 PR](https://code.launchpad.net/~chad.smith/ubuntu/+source/ubuntu-advantage-tools/+git/ubuntu-advantage-tools/+merge/389745)
- 11. For SRUs: Create an "upload a new version bug" in https://bugs.launchpad.net/ubuntu/+source/ubuntu-advantage-tools/+bugs
+ 9. Create a merge proposal such as [27.0.2 PR](https://code.launchpad.net/~chad.smith/ubuntu/+source/ubuntu-advantage-tools/+git/ubuntu-advantage-tools/+merge/402459) which targets `ubuntu/devel`
+ 10. Add 2 review slots for `canonical-server` and `canonical-server-core-reviewers`
+ 11. **For SRUs:** Create an "upload a new version bug" in https://bugs.launchpad.net/ubuntu/+source/ubuntu-advantage-tools/+bugs like [27.0](https://bugs.launchpad.net/ubuntu/+source/ubuntu-advantage-tools/+bug/1926361) or [20.3](https://bugs.launchpad.net/ubuntu/+source/ubuntu-advantage-tools/+bug/1869980)
  12. Describe the need, provide testing PPA and describe test instructions
```

Comment:
Suggest breaking the instructions and examples into sub-bullets (not actually sure if sub-bullets work though in gihub markdown)
```suggestion
 11. **For SRUs:** Create an "upload a new version bug" in https://bugs.launchpad.net/ubuntu/+source/ubuntu-advantage-tools/+bugs
     * Describe the need, provide testing PPA and describe test instructions
     * For examples, see [27.0](https://bugs.launchpad.net/ubuntu/+source/ubuntu-advantage-tools/+bug/1926361) or [20.3](https://bugs.launchpad.net/ubuntu/+source/ubuntu-advantage-tools/+bug/1869980)
```